docker/table-diff-handle/tablediffview/create_diff_markdown.py

```diff
@@ -89,8 +95,12 @@ def create_diff_rows(
             reference_row = []
 
         is_diff = create_diff_cells(diff_row, changed_row, reference_row)
-        # Only changed row will be added to diff_table, expect header row
-        if is_table_header_row(changed_row, reference_row) or is_diff:
+        # Only changed row will be added to diff_table, expect header row.
+        if (
+            not reference_row
```

Comment:
@TruongQuangSB Reference row can not be null. So no need to check for it. Or is an empty array considered as false?

Reply:
@mariusheine it is check empty array